docs/user/precheck/check_networks.rst

```diff
@@ -0,0 +1,199 @@
+configured in his Neutron configuration. Using ``vlan`` networks requires that
+any switches in user's environment are configured to trunk the corresponding
+VLANs.
+
+``physical_network`` parameter is required for creating ``vlan`` network in
+Neutron. And it also should be defined in the Neutron configuration along with
+the VLAN IDs range.  Otherwise the error will occur during the network
+migration::
+
+    Invalid input for operation: physical_network '<physnet-name>' unknown  for
+    VLAN provider network. (HTTP 400)
+
+This check verifies if physical network from the source cloud's network also
+exists on the destination cloud too.
+
+
+.. note::
+
+    Physical networks are defined in the Neutron configuration (on the server
+    side). For example:
+
+    If Neutron ML2 plugin is used on the cloud, physical networks for ``flat``
+    and ``vlan`` networks will be defined in the
+    :file:`/etc/neutron/plugins/ml2/ml2_conf.ini`::
+
+        [ml2]
+        # (ListOpt) List of network type driver entrypoints to be loaded from
+        # the neutron.ml2.type_drivers namespace.
+        #
+        type_drivers = vlan,flat
+
+        [ml2_type_flat]
+        # (ListOpt) List of physical_network names with which flat networks
+        # can be created. Use * to allow flat networks with arbitrary
+        # physical_network names.
+        #
+        flat_networks = physnet1,physnet2
+
+        [ml2_type_vlan]
+        # (ListOpt) List of <physical_network>[:<vlan_min>:<vlan_max>] tuples
+        # specifying physical_network names usable for VLAN provider and
+        # tenant networks, as well as ranges of VLAN tags on each
+        # physical_network available for allocation as tenant networks.
+        #
+        network_vlan_ranges = physnet3:300:319,physnet4:400:419,physnet5
+
+    In case of above error occurs:
+     - Compare these config files on source and destination clouds;
+     - Add missing physical networks;
+     - Reconfigure Neutron to work with these physnets.
+
+VMs spawned directly in external network
+----------------------------------------
+
+Check if there any virtual machine is spawned directly in the external networks
+on the source cloud. Some versions of OpenStack do not support such kind of
+instances creating, because it is not a proper way of external IP assigning.
+Instead of it VM should be created in the private network and use (to be
+associated with) Floating IP, allocated from external network.
+
+
+Results
+-------
+
+:dfn:`CheckNetworks` performs all above described checks. There are 2 possible
+results can be, when the task finishes:
+
+ - Check passes without any errors. It means that there are no any conflicts
+   from the networking side and migration will proceed.
+ - Check fails with the ``AbortMigrationError`` and prints list of all
+   overlappings it founds. In this case CloudFerry will not start migration and
+   operator is expected to resolve all these issues manually.
```

Comment:
According to what I read above this is not really true, there is also a chance of having warning messages during network segmentation IDs overlap stage